script.js

```diff
@@ -1,20 +1,12 @@
-const hamburger = document.querySelector(".hamburger");
-const navMenu = document.querySelector(".navMenu");
-
-hamburger.addEventListener("click", () => {
-  hamburger.classList.toggle("active");
-  navMenu.classList.toggle("active");
-})
-
-document.querySelectorAll(".nav-link").forEach(n => n.
- addEventListener("click" , () => {
-    hamburger.classList.remove("active");
-    navMenu.classList.remove("active");
- }))
-
-
- 
- 
- 
- 
- 
\ No newline at end of file
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.navMenu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```

Comment:
- [ ] when I click on the navigation links it doesn't work please make sure to when you click on one of them it should take you to the right section in order to complete the requirements